index.html

```diff
@@ -68,7 +68,340 @@ <h3>
     </div>
     </div>
     
-    </section>
+</section>
+
+<section class="recent-works" id="portfolio-section">
+   
+
+  
+  <header class="title-header">
+
+     
+      <h2 class="title">My Recent Works</h2>
+  
+<div><hr class="title-border"></div>
+    </header>
+     <div class="d-flex">
+    <div class="work">
+        <section >
+          <img src="images/bestwork.png" class="desk-img" alt="Best work">
+
+        </section>
+      </div>
+        <div class="mg">
+
+      
+        <section class="stories">
+          <h5>Multi-Post Stories</h5> 
+        </section>
+        <section class="story-details">
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+        </section>
+        <ul class="langs">
+            <li>
+                CSS
+              </li>
+            <li>
+                Html
+              </li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+        <section class="disp-btn">
+            <button class="pst-btn" id="showProject">see project</button>
+        </section>
+    </div>
+       </div>
```

Comment:
- [ ] Please note that this content (the title and the first card) of the page should be included inside the `<section class="myProjects">` since they all represent and belong to the same section , kindly fix that